.gitlab/pipelines/stages/deploy/wait.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1090
+source "$(bazel info workspace)/.gitlab/pipelines/config/config.sh"
+
+# Wait for kubecf to start.
+wait_for_router() {
+  local timeout="1800"
+  until curl -k --fail --head --silent "https://api.${SYSTEM_DOMAIN}/v2/info" || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$((timeout - 1)); done
```

Comment:
This actually waits for `router` _and_ `api`. But it's probably what we actually want.

Except it might be nice to express this in multiple lines ;)

Reply:
The router always becomes available after the api (at least with kubecf). In the next PR using kind I actually had to change this to waiting for the endpoint to become ready, which is the same as the router becoming ready.